manifest.json

```diff
@@ -60,6 +61,13 @@
                 "label": "Content Background",
                 "type": "color",
                 "default": "#FAFAFA"
+              },
+              {
+                "name": "active_index",
+                "label": "active_index",
+                "type": "string",
```

Comment:
i personally prefer indexes being ints w/ not found being -1, but this also works